services/QuillLMS/Gemfile.lock

```diff
@@ -235,19 +236,19 @@ GEM
       google-cloud-automl-v1 (~> 0.0)
       google-cloud-automl-v1beta1 (~> 0.0)
       google-cloud-core (~> 1.5)
-    google-cloud-automl-v1 (0.3.0)
+    google-cloud-automl-v1 (0.4.0)
```

Comment:
Are these google cloud updates needed?

Reply:
I'll look into this when I rebase against develop + #7770